### ISSUE TITLE
Move to traditional logit function equations

### ISSUE THREAD

**Issue by mkmahajan** [EnergyInnovation/eps-us#276]
Notes from Robbie below. We decided over email on 6/7/2023 to move to the traditional logit function approach.

I’ve been looking at some other technology models that use logit functions and I keep running into the fact their logit specification looks different than ours. For example NREL’s TEMPO model has an approach where the cost of a tech is in the exponent (i.e. instead of raising the cost to an exponent, the term takes e raised to the cost).
 
This appears to be the traditional way in which logits are formulated (e.g. [https://eml.berkeley.edu/books/choice2.html](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Feml.berkeley.edu%2Fbooks%2Fchoice2.html&data=05%7C01%7C%7Cfdba94c4e6e349a2453f08db66d82bd5%7Cc7ef1c8bba6347028e745094387a97de%7C0%7C0%7C638216850862254909%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0v%2FYZBVsC3roy5WPSo8A8mHqbJLJmj3%2FZ9vPXACHu7U%3D&reserved=0)) and standard practice in technology models. The function we have, which is from GCAM, is novel, based on my review. 
 
I am wondering if we ought to be using the more traditional logit approach, which is one of the two options GCAM uses: [https://jgcri.github.io/gcam-doc/choice.html#ref4](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fjgcri.github.io%2Fgcam-doc%2Fchoice.html%23ref4&data=05%7C01%7C%7Cfdba94c4e6e349a2453f08db66d82bd5%7Cc7ef1c8bba6347028e745094387a97de%7C0%7C0%7C638216850862254909%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=cq13h59TOfnelk%2FYNaq%2BLvdHhEeq1r5X%2BWiGv6WcR9I%3D&reserved=0).
 
The main difference in the two approaches in GCAM is that the modified logit (the one we use) changes the shape of the supply curve with a change in mean price (graph on right), where the traditional logit assumes the same distribution and just shifts costs left and right (graph on left): 
 
 
 
One of the known issues with the modified logit is that it doesn’t perform well when the price or cost value is small (note above there are values >0 for all costs for mu=2 in the modified logit, but in the unmodified logit the values go to zero).
 
I am wondering if we should use the unmodified logit instead of the modified logit because I think it will address issues related to allocating to uncompetitive technologies, distributions probably shouldn’t should shift so much, and it is more the standard practice that the modified logit, which is unique to GCAM. To elaborate a bit on the distributions: I think of differences between BEV and ICE. The main differences in vehicle cost distributions are features on cars, and I would expect the same distribution shape for ICE and BEV. Right now, we use the other approach, where distributions change as a function of the mean price change. This seems wrong. In GCAM, they say that it can be appropriate to use the modified logit when it is attempting to capture consumer preferences that they don’t model, but we try to model some of these + I don’t think that rationale holds up with consumer choices. 
 
It’s not that big a deal to update these in the model, but I am thinking it will help address some weirdness + align us with standard modeling practice.
 
Two final thoughts. First, changing this approach makes the model much more sensitive to price. You can see the difference by looking at the area under the purple curve at cost = 1.5, which is effectively zero in the first approach but about 1/3 in the second approach. Second, one of the reasons we chose the modified logit is that it is easier to calibrate, so I’d want to test this out in Excel and how we would calibrate it before making the shift, but generally am supportive of making this change.

**Comment by mkmahajan** [comment]
Robbie completed this issue.
